src/EnergyPlus/DemandManager.cc

```diff
@@ -632,8 +638,15 @@ namespace DemandManager {
 			MaxAlphas = max( MaxAlphas, NumAlphas );
 			MaxNums = max( MaxNums, NumNums );
 		}
+		CurrentModuleObject = "DemandManager:Ventilation";
+		NumDemandMgrVentilation = GetNumObjectsFound(CurrentModuleObject);
+		if (NumDemandMgrVentilation > 0) {
```

Comment:
@vidanovic  Please check all your codes to follow the EnergyPlus coding style: (1) put a space between braces and the arguments.